### PR TITLE
Allclear 364

### PR DESCRIPTION
**Pull request by SRSaddler**

### CODE REVIEW

File: src/components/general/inputs/phone-number-input.js [hunk 1]
```diff
@@ -56,9 +56,9 @@ class PhoneNumberInput extends Component {
   }
 
   onKeyDown(evt) {
-    // todo: for enter key
+    // todo: for enter key, investigate
     if(evt.key === 'Enter'){
-      this.props.onSubmit(this.state.phone);
+      //this.props.onSubmit(this.state.phone);
```
Comment (crimsonsunset) [inline src/components/general/inputs/phone-number-input.js:61]: whats wrong with this? seems to be working fine for me and there was another ticket that would be undone by commenting it out

File: src/pages/sign-in-verification.page.js [hunk 2]
```diff
@@ -149,8 +200,16 @@ export default class SignInVerificationPage extends Component {
...
                   />
                 </FormControl>
+              </div>
 
+              <div className="alert-messages">
                 {this.state.error === true ? <p className="error">{this.state.message}</p> : ''}
+                {this.state.smsTimeoutEnabled === true ? <p className="no-code-message">Didnt receive a code?</p> : ''}
+                {this.state.smsTimeoutEnabled === true ?
+                  <Button onClick={() => this.resendCode()} variant="contained" className="back">
+                    Resend Code
```
Comment (crimsonsunset) [inline src/pages/sign-in-verification.page.js:210]: could probably do this a bit more elgantly w a snackbar

File: src/pages/sign-up-verification.page.js [hunk 3]
```diff
@@ -29,16 +29,43 @@ export default class SignUpVerificationPage extends Component {
     this.state = {
```
Comment (crimsonsunset) [inline src/pages/sign-up-verification.page.js:29]: in general im a little unclear why signUp and signIn both need seperate pages. cant we combine them and make a few minor checks?

File: src/components/general/inputs/phone-number-input.js (hunk 1 above)
Comment (SRSaddler) [reply src/components/general/inputs/phone-number-input.js:61]: It broke, wanted to unblock for launch. It doesn't work for me. I'll try again

File: src/pages/sign-up-verification.page.js (hunk 3 above)
Comment (SRSaddler) [reply src/pages/sign-up-verification.page.js:29]: Yeah can happen later...priority is to cut a prod tag and not rework it..I'll add it to the improvements list with Kev

**Pull request merged**